ci/terraform/oidc/api-gateway.tf

```diff
@@ -584,16 +586,27 @@ resource "aws_api_gateway_resource" "orch_frontend_resource" {
   path_part   = "orch-frontend"
 }
 
-resource "aws_api_gateway_method" "orch_frontend_method" {
+resource "aws_api_gateway_resource" "orch_frontend_resource_proxy" {
   count       = var.orch_frontend_api_gateway_integration_enabled ? 1 : 0
   rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
-  resource_id = aws_api_gateway_resource.orch_frontend_resource[0].id
-  http_method = "GET"
+  parent_id   = aws_api_gateway_resource.orch_frontend_resource[0].id
+  path_part   = "{proxy+}"
+}
+
+resource "aws_api_gateway_method" "orch_frontend_proxy_method" {
+  count       = var.orch_frontend_api_gateway_integration_enabled ? 1 : 0
+  rest_api_id = aws_api_gateway_rest_api.di_authentication_api.id
+  resource_id = aws_api_gateway_resource.orch_frontend_resource_proxy[0].id
+  http_method = "ANY"
 
   depends_on = [
-    aws_api_gateway_resource.orch_frontend_resource
+    aws_api_gateway_resource.orch_frontend_resource_proxy
```

Comment:
Is this right, or should it be `...proxy[0]`? I'm not sure

Reply:
I'm not totally convinced these depends on do anything anyway, since the resource is referenced and terraform is smart enough to work it out

Reply:
Yeah googling it definitely makes me think it's unnecessary